libr/main/radiff2.c

```diff
@@ -140,14 +140,15 @@ static int cb(RDiff *d, void *user, RDiffOp *op) {
 				printf ("-0x%08"PFMT64x":", op->a_off);
 				int len = op->a_len; // R_MIN (op->a_len, strlen (op->a_buf));
 				for (i = 0; i < len; i++) {
-					printf ("%02x", op->a_buf[i]);
+					printf ("%02x ", op->a_buf[i]);
 				}
-				char *p = r_str_escape ((const char*)op->a_buf);
-				printf (" \"%s\"\n", p);
-				free (p);
 				if (!quiet) {
+					char *p = r_str_escape ((const char*)op->a_buf);
+					printf (" \"%s\"", p);
```

Comment:
maybe worth to chk if  is not null

Reply:
There is a check before that already

https://github.com/radareorg/radare2/blob/778c42cfa7b9783b39aeda52993fa6b4fc7161f1/libr/main/radiff2.c#L154-L155